backend/app/routes/routes.py

```
@@ -59,19 +68,81 @@ async def home():
+                return obj.model_dump()
+            if hasattr(obj, "dict"):  # Pydantic v1
+                return obj.dict()
+            return str(obj)
+        
+        # Save response as formatted JSON
+        json_content = json.dumps(response, indent=2, ensure_ascii=False, default=json_serializer)
+        await asyncio.to_thread(
+            lambda: filepath.write_text(json_content, encoding="utf-8")
+        )
+        logger.info(f"Saved cache response to: {filepath}")
+    except Exception as e:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)